src/com/esotericsoftware/kryo/util/MapReferenceResolver.java

```diff
@@ -30,18 +30,21 @@
  * for get or put.
  * @author Nathan Sweet */
 public class MapReferenceResolver implements ReferenceResolver {
-	private static final int CAPACITY = 2048;
-	private static final int MAXIMUM_CAPACITY = 8192;

```

Comment:
#705 introduced a `maximumCapacity` that only applies for `readObjects`. I decided to replace this with a single, customizable value that controls the maximum capacity of both data structures.